src/03_video_nodes.js

```diff
@@ -300,6 +300,14 @@ Class ("paella.FlashVideo", paella.VideoElementBase,{
 	},
 
 	processEvent:function(eventName,params) {
+		// #DCE MATT-1355, Live player mute button shows muted when live stream is not muted
```

Comment:
I would much rather this be implemented as an evented plugin that only functions when we're doing a live stream, it seems like this code will run for every single event, right?

An evented plugin could be harnessed similar to this one: https://github.com/harvard-dce/paella/blob/dce-release/vendor/plugins/edu.harvard.dce.paella.resizeLiveStream/resizeLiveStream.js

My concern is just that we're patching the core yet again, and those patches don't go away even when the functionality is fixed upstream - it's super hard to link our patches with upstream ones.